selenium_tests/course_info/page_objects/course_info_base_page_object.py

```diff
@@ -1,15 +1,7 @@
-from selenium.common.exceptions import NoSuchElementException
 from selenium_common.base_page_object import BasePageObject
 
 
 class CourseInfoBasePageObject(BasePageObject):
-    """
-    This is the base page object class that all Course Info pages can inherit from
-    Locators and Services would be common to all pages on  this tool
-    """
     def __init__(self, driver):
         super(CourseInfoBasePageObject, self).__init__(driver)
-        try:
-            self._driver.switch_to.frame(self._driver.find_element_by_id("tool_content"))
-        except NoSuchElementException:
-            pass
+        self.focus_on_tool_frame()
```

Comment:
This seems like a candidate for moving into selenium common (as `BaseLTIPageObject`, maybe) in the future.